config/config.go

```diff
@@ -35,6 +37,7 @@ func Get() (*Config, error) {
 		PostgresURL:             "user=dp dbname=FilterJobs sslmode=disable",
 		SecretKey:               "FD0108EA-825D-411C-9B1D-41EF7727F465",
 		ShutdownTimeout:         5 * time.Second,
+		DatasetAPIURL:           "http://localhost:22000",
```

Comment:
Do we want a default `DatasetAPIAuthToken` set? I understand that if it changes in the dataset API we would have to change it here to.

I guess another argument is it forces the user of the service to set their own auth tokens instead of relying on insecure defaults.

Reply:
we don't want a default set, for security reasons (github, etc), but we have decided to have one for now (dev convenience) - we will remove later - I have added the default